test/unit/specs/scripts/extension-utils.spec.js

```diff
@@ -75,6 +75,20 @@ describe(`Extension Utils`, () => {
         }
       ])
     })
+
+    it("should do nothing with no message type", () => {
+      const result = processLunieExtensionMessages(store)({
+        source: global,
+        data: {
+          type: "NOT_FROM_LUNIE_EXTENSION",
```

Comment:
```suggestion
          type: "FROM_LUNIE_EXTENSION",
```
This would get filtered out by `filterExtensionMessage`

Reply:
🤦‍♂  Derp. Thanks!